saleor/account/forms.py

```diff
@@ -127,3 +127,27 @@ def send_mail(
         # template, we remove it from the context.
         del context['user']
         emails.send_password_reset_email.delay(context, to_email)
+
+
+class EmailChangeForm(forms.Form):
```

Comment:
Can we use `ModelForm` here? It will simplify things out, passing user to the constructor won't be necessary, as we could take it from the instance